### PR TITLE
Verify 'anagram' - 'beer-song' exercise versions

### PR DESCRIPTION
**Pull request by filalex77**
- [x] anagram
- [x] armstrong-numbers
- [x] atbash-cipher
- [x] bank-account: doesn't have `canonical-data.json`; put at `1.0.0`
- [x] beer-song

Related to #510. 

r? @neenjaw 

---
This change is [Reviewable](https://reviewable.io/reviews/exercism/elixir/512)

### CODE REVIEW

File: exercises/beer-song/test/beer_song_test.exs [hunk 1]
```diff
@@ -1,356 +1,381 @@
 defmodule BeerSongTest do
   use ExUnit.Case
 
-  test "getting the first verse (99 bottles)" do
-    assert BeerSong.verse(99) == """
-           99 bottles of beer on the wall, 99 bottles of beer.
-           Take one down and pass it around, 98 bottles of beer on the wall.
-           """
+  describe "verse" do
+    describe "single verse" do
```
Comment (devonestes) [inline exercises/beer-song/test/beer_song_test.exs]: Nested describes are not supported in Elixir, so you'll need to find another way to set this up.

Comment (filalex77) [reply exercises/beer-song/test/beer_song_test.exs]: Thanks for pointing out! I think just dropping the second level is okay.

File: exercises/bank-account/.meta/version [hunk 2]
```diff
@@ -1 +1 @@
-0.0.0
+1.0.0
```
Comment (neenjaw) [inline exercises/bank-account/.meta/version]:
```suggestion
0.0.0
```

For tests that don't have data, we are going to leave it at `0.0.0` like the JavaScript and TypeScript group. I'll clarify in the issue.

Comment (filalex77) [reply exercises/bank-account/.meta/version]: Sure, thanks for pointing out.

**Pull request merged**